From 4d0b6e3979aaf4bf80b477c73a63ba6339f529d9 Mon Sep 17 00:00:00 2001
From: rleix <rayx.lei@intel.com>
Date: Mon, 26 Jun 2017 11:23:43 +0800
Subject: [PATCH] Remove the configuration of SettingsActivity from
 CarLatinIME.

As there is no available SettingsActivity in CarLatinIME. Remove it from configuration xml file.

Change-Id: I3f1c7bdd8ab0505a37e439e91f564a9bbfe49e35
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-45853
Signed-off-by: Lei,RayX <rayx.lei@intel.com>
---
 res/xml/method.xml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/res/xml/method.xml b/res/xml/method.xml
index 5de859b..2f0fc02 100644
--- a/res/xml/method.xml
+++ b/res/xml/method.xml
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <input-method xmlns:android="http://schemas.android.com/apk/res/android"
-    android:settingsActivity="com.android.inputmethod.latin.settings.SettingsActivity"
     android:isDefault="true"
     android:supportsSwitchingToNextInputMethod="false">
     <subtype android:icon="@drawable/ic_ime_switcher_dark"
@@ -10,4 +9,4 @@
         android:languageTag="en-US"
         android:imeSubtypeMode="keyboard"
         android:isAsciiCapable="true" />
-</input-method>
\ No newline at end of file
+</input-method>
-- 
1.9.1

